website/src/docs/statusbar.md

```diff
@@ -12,6 +12,39 @@ Best used together with a simple file source plugin, such as [FileInput][] or [D
 
 ## Options
 
+```js
```

Comment:
This could perhaps use an: "All of the options, and their defaults" intro?

Reply:
this is how it's currently done on all plugin pages, but yes you're right that it could use an intro sentence

Reply:
ah okay, that helps a lot already